Sensus.Android/Callbacks/AndroidNotifier.cs

```diff
@@ -55,7 +55,19 @@ public override void IssueNotificationAsync(string title, string message, string
                         .SetAutoCancel(true)
                         .SetOngoing(false);
 
-                    if (playSound)
+                    // if a protocol ID has been passed to the method, check the protocol's Notification Alert Exclusion Windows
+                    // to determine whether to vibrate and play sound. if protocolId is null, just use the vibrateAndPlaySound parameter.
+                    if (protocolId != null)
+                    {
+                        var runningProtocol = SensusServiceHelper.Get().GetRunningProtocolById(protocolId);
+
+                        if (runningProtocol != null)
+                            foreach (Window window in runningProtocol.NotificationAlertExclusionWindowsList)
+                                if (window.EncompassesCurrentTime())
+                                    vibrateAndPlaySound = false;
+                    }
```

Comment:
I think the above code can be pushed down to `Sensus.Callbacks.Notifier`, eliminating redundancy across the three concrete Notifier classes (1 Android and 2 iOS).